preppal-fe/src/pages/profile.tsx

```diff
@@ -0,0 +1,89 @@
+import Card from 'react-bootstrap/Card';
+import Image from 'react-bootstrap/Image';
+import { Button, Col, Row, Stack } from 'react-bootstrap';
+import React from 'react';
+
+const logo = require('../assets/logo.png')
+
+function Profile() {
+    const [userName, setName] = React.useState("");
+    const [userBio, setBio] = React.useState("");
+    const [userFollowing, setFollowing] = React.useState<any[]>([]);
+    const [userFollowingCount, setFollowingCount] = React.useState(0);
+    React.useEffect(() => {
+        fillUserContent();
+    }, []);
```

Comment:
might need to add `// eslint-disable-next-line` before this line

Reply:
Added requested line, ran 'npm run build' and 'npm start' to confirm working.